satellite/satellitedb/migrate.go

```diff
@@ -407,18 +409,25 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 						inline bigint NOT NULL,
 						allocated bigint NOT NULL,
 						settled bigint NOT NULL,
-						PRIMARY KEY ( bucket_id ),
-						UNIQUE ( bucket_id, interval_start, interval_seconds, action )
+						PRIMARY KEY ( bucket_id, interval_start, action )
+					)`,
+					`CREATE INDEX bucket_id_interval_start_interval_seconds_index ON bucket_bandwidth_rollups (
+						bucket_id,
+						interval_start,
+						interval_seconds
 					)`,
 					`CREATE TABLE bucket_storage_rollups (
 						bucket_id bytea NOT NULL,
 						interval_start timestamp NOT NULL,
 						interval_seconds integer NOT NULL,
 						inline bigint NOT NULL,
 						remote bigint NOT NULL,
-						PRIMARY KEY ( bucket_id ),
-						UNIQUE ( bucket_id, interval_start, interval_seconds )
+						PRIMARY KEY ( bucket_id, interval_start )
 					)`,
+					`ALTER TABLE bucket_usages DROP CONSTRAINT bucket_usages_rollup_end_time_bucket_id_key`,
+					`CREATE UNIQUE INDEX IF NOT EXISTS bucket_id_rollup_end_time_index ON bucket_usages ( 
```

Comment:
better not use `IF NOT EXISTS` because we might miss some states that we didn't cover e.g. this index existed for some reason and we won't know about it during migration.

Reply:
Done